### PR TITLE
Added Turkish

### PR DESCRIPTION
**Pull request by udokmeci**
PluralSpecs and test
tested OK.

### CODE REVIEW

File: i18n/language/pluralspec.go [hunk 1]
```diff
@@ -339,6 +339,17 @@ var pluralSpecs = map[string]*PluralSpec{
 		},
 	},
 
+	// Turkish
+	"tr": &PluralSpec{
+		Plurals: newPluralSet(Zero, Other),
```
Comment (nicksnyder) [inline i18n/language/pluralspec.go]: This logic doesn't seem consistent with:
http://www.unicode.org/cldr/charts/latest/supplemental/language_plural_rules.html

Can you double check this?

**Comment by udokmeci** [comment]
Ok now it fits

**Comment by nicksnyder** [comment]
I still don't think this logic is correct. You haven't tested numbers like "1.0", "1.000" or "1.00001".

Please look at the implementation and tests for Spanish; they should be identical for Turkish.

**Comment by udokmeci** [comment]
Sorry for too much try. I used right operands this time and added the ones you mention to tests.

**Pull request merged**